src/content/1.3/code/scala/snippet02.scala

```diff
@@ -0,0 +1,6 @@
+object Monoid {
+  implicit def stringMonoid: Monoid[String] = new Monoid[String] {
+    def mempty: String = ""
```

Comment:
Since the `Monoid` for **String** doesn't require any implicit evidence to be constructed - it may be a _implicit val_ instead of an _implicit def_.
Also, the empty value may be a _val_ too.